dev-qt/qtmultimedia/qtmultimedia-5.15.2.9999.ebuild

```diff
@@ -33,6 +33,7 @@ RDEPEND="
 	widgets? (
 		=dev-qt/qtopengl-${QT5_PV}*
 		=dev-qt/qtwidgets-${QT5_PV}*[gles2-only=]
+		media-libs/libglvnd
```

Comment:
I'm not sure about this one, I don't see a direct dependency, feels like a transitive dependency from the OpenGLExtensions module (part of qtgui)

Reply:
At least in my local qtmultimedia build with USE="gstreamer pulseaudio` I only see these libraries in `NEEDED` of `objdump -d` on the shared objects:

````
  NEEDED               libc.so.6
  NEEDED               libglib-2.0.so.0
  NEEDED               libgobject-2.0.so.0
  NEEDED               libgstapp-1.0.so.0
  NEEDED               libgstaudio-1.0.so.0
  NEEDED               libgstbase-1.0.so.0
  NEEDED               libgstgl-1.0.so.0
  NEEDED               libgstpbutils-1.0.so.0
  NEEDED               libgstphotography-1.0.so.0
  NEEDED               libgstreamer-1.0.so.0
  NEEDED               libgstvideo-1.0.so.0
  NEEDED               libm.so.6
  NEEDED               libpthread.so.0
  NEEDED               libpulse.so.0
  NEEDED               libQt5Core.so.5
  NEEDED               libQt5Gui.so.5
  NEEDED               libQt5MultimediaGstTools.so.5
  NEEDED               libQt5Multimedia.so.5
  NEEDED               libQt5Network.so.5
  NEEDED               libstdc++.so.6
````

so I think the dependency on libglvnd might not be needed. I didn't check whether qtmultimedia uses anything from libglvnd at build-time though, so I'm not sure one way or another.

Reply:
At least lddtree lists libGL.so:
```
libQt5MultimediaWidgets.so => /usr/lib64/libQt5MultimediaWidgets.so
...
    libGL.so.1 => /usr/lib64/libGL.so.1
        libGLdispatch.so.0 => /usr/lib64/libGLdispatch.so.0
        libGLX.so.0 => /usr/lib64/libGLX.so.0
            libX11.so.6 => /usr/lib64/libX11.so.6
```

Reply:
> I'm not sure about this one, I don't see a direct dependency, feels like a transitive dependency from the OpenGLExtensions module (part of qtgui)

@Pesa: If this is the case then I can drop that change again, and have `dev-qt/qtgui` continue enforcing the dependency anyway.

@mattst88: You would need to emerge qtmultimedia with IUSE=widgets.

Reply:
I tested locally and it does show up in DT_NEEDED, so I'd say adding the dependency is correct.